common/xfwm4/assets-dark.svg

```diff
@@ -1707,7 +1707,7 @@
     <rect
        style="opacity:1;fill:#21252b;fill-opacity:1;stroke:none;stroke-width:1;stroke-linecap:butt;stroke-linejoin:round;stroke-miterlimit:4;stroke-dasharray:none;stroke-dashoffset:0;stroke-opacity:1"
        id="bottom-inactive"
-       width="2.9999716"
+       width="1"
```

Comment:
The comment states this is 2px, but the value is 1. Is this correct?

Reply:
Good catch, I don't know how that happened... Anyway, fixed now